### PR TITLE
Lanche

### PR DESCRIPTION
**Pull request by Ore-stack**
Adding project to the repo

### CODE REVIEW

**Review by tofetpuzo — COMMENTED** [review]
You deleted all the files, I think you might have to fork the repo again.

File: .gitignore
Comment (tofetpuzo) [inline .gitignore:1]: you deleted this too why?

File: CONTRIBUTE.md
Comment (tofetpuzo) [inline CONTRIBUTE.md:1]: you might have to reclone the repo

File: .gitignore
Comment (Ore-stack) [reply .gitignore:1]: I think I deleted cause I felt I haven't done the right thing

**Pull request closed without merging**